plugins/main/public/components/security/policies/create-policy.tsx

```diff
@@ -171,7 +171,7 @@ export const CreatePolicyFlyout = ({ closeFlyout }) => {
           effect: effectValue,
         },
       });
-      const resultData = (result.data || {}).data;
+      const resultData = result?.data?.data;
       if (resultData.failed_items && resultData.failed_items.length) {
```

Comment:
If resultData is undefined, there may be an error trying to read failed_items from undefined. Right?

Reply:
Done